cmd/kogito/command/shared/install_services.go

```diff
@@ -54,6 +55,9 @@ type ServicesInstallation interface {
 	// InstallDataIndex installs Data Index. If no reference provided, it will install the default instance.
 	// Depends on the Operator, install it first.
 	InstallDataIndex(dataIndex *v1alpha1.KogitoDataIndex) ServicesInstallation
+
```

Comment:
maybe add a comment like the other methods with more details about Trusty service installation?

Reply:
Done :+1: